src/ui/theme/components/text.ts

```diff
@@ -0,0 +1,24 @@
+const TextStyle = {
+  // style object for base or default style
+  baseStyle: {},
+  // styles for different sizes ("sm", "md", "lg")
+  sizes: {},
+  // styles for different visual variants ("outline", "solid")
+  variants: {
+    headerNav: variantHeaderNav,
+  },
+  // default values for `size`, `variant`, `colorScheme`
+  defaultProps: {},
+};
+
+function variantHeaderNav() {
```

Comment:
why have this defined here rather than in styled components?  Why use this only for "settings" and "toggle fullscreen" but not the "Return to NYPL" which visually looks the same?  Is this a Chakra best practice?

Reply:
This is also used for **Return to NYPL**, it's a `<Text />` variant. I created the variant b/c it's being used in multiple places.